docs/src/components/Layout/GlobalNav/components/icons/ExternalLink.tsx

```diff
@@ -0,0 +1,19 @@
+import { Icon, Flex } from '@aws-amplify/ui-react';
+import styles from '../../GlobalNav.module.scss';
+
+export function ExternalLink({ children }: { children: any }) {
+  return (
+    <Flex alignItems="center" gap="xs">
+      {children}
+      <Icon ariaLabel="External link" viewBox={{ width: 18, height: 18 }}>
```

Comment:
Should it be the same size as the IconLinks?

![image](https://user-images.githubusercontent.com/11983489/193651411-72736cf4-985b-4402-a375-2d9a4287492a.png)

They are not aligned